internal/test/test.go

```diff
@@ -148,6 +149,9 @@ func testCode(
 ) (*result, error) {
 	runner := cdcTests.NewTestRunner()
 
+	logger := zerolog.New(zerolog.ConsoleWriter{Out: os.Stderr}).With().Timestamp().Logger()
```

Comment:
Can't you just put `logger` above `runner` then do `runner := cdcTests.NewTestRunner().WithLogger(logger)`?